docs/README.md

```diff
@@ -0,0 +1,388 @@
+To check that the SDK methods are returning a correct type of an instance, they will all contain an `entityType` in their response. `enityType` can be one of four values:
+`Variable`, `User`, `Feature`, `Object`, `Void`
+
+### How to use
+Before calling any SDK methods, you must first create an instance of the SDK client. This is done by calling the `/client` endpoint with the `POST` method. The response will contain a `Location` header
+which can be used to reference the client instance in subsequent calls.
+
+The next step is to create a user instance. This is done by calling the `/user` endpoint with the `POST` method. The response will contain a `Location` header which can be used to reference the user instance in subsequent calls.
+
+Once you have a client and user instance, you can invoke any SDK method by calling the `/{location}/command` endpoint with the `POST` method.
+
+<h1 id="devcycle-test-harness-proxy-client">Client</h1>
+
+SDK Client
+
+<a id="opIdcreateClient"></a>
+
+`POST /client`
+
+*Create a DVC client instance*
+
+> Body parameter
+
+```json
+{
+  "clientId": "string",
+  "sdkKey": "string",
+  "options": {},
+  "enableCloudBucketing": true
+}
+```
+
+<h3 id="createclient-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|`clientId`|body|string|true|A unique ID that will be used to reference the client once it has been created|
+|`sdkKey`|body|string|false|A unique key used to initialize the DVC SDK|
+|`options`|body|object|false|An object of launch parameters. See [DevCycle Docs](https://docs.devcycle.com/docs/sdk/server-side-sdks/node#initialization-options) for a full list of options|
+|`enableCloudBucketing`|body|boolean|false|An option to use the cloud bucketing version of the SDK|
+
+> Example responses
+
+> 200 Response
+
+```json
+{
+  "exception": "Missing environment key! Call initialize with a valid environment key"
+}
+```
+
+> 201 Response
+
+```json
+{
+  "message": "Success"
+}
+```
+
+> 400 Response
+
+```json
+{
+  "error": "Invalid request: missing clientId"
+}
+```
+
+<h3 id="createclient-responses">Responses</h3>
+
+|Status|Meaning|Description|
+|---|---|---|
+|200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|
+|201|[Created](https://tools.ietf.org/html/rfc7231#section-6.3.2)|Created|
+|400|[Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1)|Bad Request|
+
+<h3 id="createclient-responseschema">Response Schema</h3>
+
+Status Code **200**
+
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`» exception`|string|false|An error from the SDK indicating an exception was thrown during initialization|
+
+Status Code **201**
+
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`» message`|string|false|A message indicating the result of initializing an SDK client|
+
+Status Code **400**
+
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`» error`|string|true|An error message indicating what went wrong with the request|
```

Comment:
this field is called "message" still

Reply:
looking at the implementation, when we return a 400, it has `error` instead of `message`

Reply:
oh nvm I was looking at the wrong thing. Will fix